tests/functional/flakes/flakes.sh

```diff
@@ -285,6 +285,35 @@ git -C "$flake3Dir" add flake.lock
 
 git -C "$flake3Dir" commit -m 'Add lockfile'
 
+# Test accessing output in installables with `.` (foobarbaz.<output>)
+cat > "$flake3Dir/flake.nix" <<EOF
+{
+    outputs = {self}: {
+      packages.$system.hello = (import ./config.nix).mkDerivation {
+        name = "hello";
+        outputs = [ "foo" "out" ];
+        meta.outputsToInstall = [ "out" ];
+        buildCommand = ''
+            mkdir \$foo \$out
+            echo "foo" > \$foo/file
+            echo "out" > \$out/file
+        '';
+        outputSpecified = true;
+      };
+    };
+}
+EOF
+
+cp ../config.nix "$flake3Dir"
+git -C "$flake3Dir" add flake.nix config.nix
+git -C "$flake3Dir" commit -m 'multi outputs flake'
+
```

Comment:
[`--no-link`](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-build#opt-no-link) flag shouldn't produce any result i think, 
- and since we are just testing for output selection, i took a bit of inspiration from [here](https://github.com/NixOS/nix/blob/189e40214e450ba1ffa7b5bb4e3f8dd29d7e6982/tests/functional/build.sh#L21)

Reply:
Oops, I read that too quickly.
I guess `--no-link` is ok, assuming the code is sufficiently orthogonal on the inside.